README.md

```diff
@@ -65,6 +65,45 @@ app('api.router')->version(config('api.version'), ['namespace' => 'Saritasa\Lara
   });
 ```  
 
+```php
+<?php
+
+class LoginRequest extends FormRequest implements ILoginRequest
+{
+    /**
+     * Allow all users access here.
+     * 
+     * @return bool
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function rules(): array
+    {
+        return [
+            'email' => 'required|string',
```

Comment:
These credentials are exactly like in library, so this example makes no sense. Feature is to override them - show example that overrirdes default.

What exactly different in your application from default request?
email -> username?
password -> token?

Reply:
done

Reply:
updated